### PR TITLE
Add RC message

### PR DESCRIPTION
**Pull request by caleballdrin**
## Description

- Add message telling users that Responsibility Centers is not working.

## Checklist:

- [ ] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [ ] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [ ] I have requested a review from another person on the project

### CODE REVIEW

File: src/components/Reports/ResponsibilityCentersReport/ResponsibilityCentersReport.tsx [hunk 1]
```diff
@@ -207,6 +213,11 @@ export const ResponsibilityCentersReport: React.FC<Props> = ({
         />
       ) : (
         <ScrollBox data-testid="ResponsibilityCentersScrollBox">
+          <Alert severity="warning">
+            {t(
+              'Responsibility Centers is still in development. Some features are not working. Please check back later.',
```
Comment (dr-bizz) [inline src/components/Reports/ResponsibilityCentersReport/ResponsibilityCentersReport.tsx:218]: I would say it should be coming in the next few days to give people hope. Sometimes Companies say a feature is coming soon and it takes weeks/months.

**Review by dr-bizz — APPROVED** [review]
Looos great. I had one suggestion but will approve.

File: src/components/Reports/ResponsibilityCentersReport/ResponsibilityCentersReport.tsx [hunk 2]
```diff
@@ -215,7 +215,7 @@ export const ResponsibilityCentersReport: React.FC<Props> = ({
         <ScrollBox data-testid="ResponsibilityCentersScrollBox">
           <Alert severity="warning">
             {t(
-              'Responsibility Centers is still in development. Some features are not working. Please check back later.',
+              'Responsibility Centers is still in development and may not be complete until October 11th. Please check back later.',
```
[the remark this answers is not in the document]
Comment (caleballdrin) [reply src/components/Reports/ResponsibilityCentersReport/ResponsibilityCentersReport.tsx:218]: What do you think it should say?

[the remark this answers is not in the document]
Comment (wrandall22) [reply src/components/Reports/ResponsibilityCentersReport/ResponsibilityCentersReport.tsx:218]: a few days, like Daniel said, seems sufficient.

**Pull request merged**